Common/Systems/Affixes/ItemTypes/DefenseAffixes.cs

```diff
@@ -38,4 +40,12 @@ public override void ApplyAffix(Player player, EntityModifier modifier, Item ite
 	{
 		modifier.ReflectedDamageModifier += Value;
 	}
+}
+
+internal class MultiplyBlockAffix : ItemAffix
```

Comment:
I think this should be called `IncreasedBlockChanceAffix`
In reality at the end of the day it's multiplying by a divised number so multiply makes sense, but increased keep it inline with the rest of the naming

Reply:
Functionally the passive and the affix should be no different. So having a different name seems weird.